### DISCUSSION TITLE
How to use Budibase

### DISCUSSION THREAD

**Discussion by ARAFATHYASIN** [Budibase/budibase#7040] [Help] [answered]
Dear Team,

Kindly help me, I am new to budibase I do no who to use this I create a template and I do know how to write the code and how to host in the online

waiting for the response

**Comment by ghost** [comment]
Have you checked out the docs? 
https://docs.budibase.com/docs

**Comment by ARAFATHYASIN** [comment]
Dear Team,
i create the webapp and everything is done,Kindly have a look on below
screenshot


I do no how to publish in online,Know its running in Localhost

Kindly explain to me how to Deploy the Web Application online by briefly.

**Comment by ghost** [comment, marked as answer]
I'm not a developer unfortunately, so the only guidance I can give it to check out their docs - https://docs.budibase.com/docs/hosting-settings

**Reply by melohagan** [reply]
Hey @ARAFATHYASIN 

If you're not comfortable with self hosting, it may be a good idea to setup a cloud account. Budibase can handle the hosting for you!
You can read more in the linked docs page - thanks @adamjdavies !
